### ISSUE TITLE
About the loss change?

### ISSUE THREAD

**Issue by idealboy** [XuecaiHu/Meta-SR-Pytorch#14]
sir,

what is the final loss when train on DIV2K?

I have the loss~4.0, and it swings around 4.0 when training, is it normal?

thank you very much

**Comment by idealboy** [comment]
And, I use the command to train:

python main.py --model metardn --save metardn --ext sep --lr_decay 200 --epochs 1000 --n_GPUs 4 --batch_size 16 

with all other parameters default in the option.py

thank you!

**Comment by XuecaiHu** [comment]
it is normal
